org-formation/090-systems-manager/Scheduled-Script-Automation.yaml

```diff
@@ -0,0 +1,297 @@
+            TargetAccounts:
+              Fn::Join:
+                - ","
+                - - Fn::EnumTargetAccounts ScriptExecutionMemberBinding ${account}
+            TargetRegionIds: !Ref TargetRegionIds
+            TargetLocationMaxConcurrency: !Ref TargetLocationMaxConcurrency
+            TargetLocationMaxErrors: !Ref TargetLocationMaxErrors
+            ExecutionRoleName: !Ref ExecutionRoleName
+            MasterAccountID: !Sub ${AWS::AccountId}
+            AutomationDocumentScriptExecution: !Ref AutomationDocumentScriptExecution
+            ScriptUrl: !Ref ScriptUrl
+            ScriptParameters: !Ref ScriptParameters
+            TargetTagName: !Ref TargetTagName
+            TargetTagValue: !Ref TargetTagValue
+      FunctionName: MultiAccountScriptExecution
+      Handler: index.handler
+      Role: !GetAtt AWSLambdaSSMMultiAccountScriptExecutionRole.Arn
+      Runtime: python3.12
+  AutomationAdministrationServiceRole:
+    Type: AWS::IAM::Role
+    Condition: CreateAutomationAdministrationRoleCondition
+    Properties:
+      RoleName: AWS-SystemsManager-AutomationAdministrationRole
+      AssumeRolePolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+        - Effect: Allow
+          Principal:
+            Service: ssm.amazonaws.com
+          Action:
+          - sts:AssumeRole
+      Path: "/"
+      Policies:
+      - PolicyName: AssumeRole-AWSSystemsManagerAutomationExecutionRole
+        PolicyDocument:
+          Version: '2012-10-17'
+          Statement:
+          - Effect: Allow
+            Action:
+            - sts:AssumeRole
+            Resource:
+              Fn::Sub: arn:${AWS::Partition}:iam::*:role/AWS-SystemsManager-AutomationExecutionRole
+          - Effect: Allow
+            Action:
+            - organizations:ListAccountsForParent
+            Resource:
+            - "*"
+  AutomationDocumentScriptExecution:
+    Type: "AWS::SSM::Document"
+    Properties:
+      DocumentType: "Command"
+      Name: "Automation-Run-Script"
+      Content:
+        schemaVersion: "2.2"
+        description: "Run a parameterized script on EC2 instances"
+        parameters:
+          ScriptUrl:
+            type: String
+            description: "The URL of the script to run"
+          SsmParameterNameToEnvvarMap:
+            type: String
+            description: >-
+              space delimited list of environment name-value pairs in the format name:value,
+              e.g., ENVNAME1:envval1 ENVNAME2:envval2
+          TargetTagName:
+            type: String
+            description: name of the tag marking instances on which to run the script
+          TargetTagValue:
+            type: String
+            description: value of the tag marking instances on which to run the script
+        mainSteps:
+        - name: "runShellScript"
+          action: "aws:runShellScript"
+          timeoutSeconds: 7200
```

Comment:
this 2hr timeout makes sense when running OS patches however it seems excessive when installing the nessus agent.  consider parameterizing this timeout?